### PR TITLE
tests: Add tests for copy, verify, dsm commands

### PR DESCRIPTION
**Pull request by Arunpandian15**
Tests has been added for copy, verify, DSM
commands and respective tests updated in meson.build
file.

Signed-off-by: Arunpandian J <apj.arun@samsung.com>

### CODE REVIEW

File: tests/nvme_copy_test.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved.
+#
```
Comment (igaw) [inline tests/nvme_copy_test.py:2]: I think we should start using SPDX licensee identifier for all files and remove the license text. Would you mind submitting these new files with a SPDX tag?

The main motivation is to make any tooling around tool chain verification simpler.

**Review by igaw — CHANGES_REQUESTED** [review]
Looks good, except my request on adding SPDX tags and removing the license boilerplate text.

File: tests/nvme_copy_test.py (hunk 1 above)
Comment (Arunpandian15) [reply tests/nvme_copy_test.py:2]: @igaw you mean to say by adding "SPDX-License-Identifier: LGPL-2.1-or-later" this text and remove "Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved." this one right ?

File: tests/nvme_copy_test.py [hunk 2]
```diff
@@ -1,17 +1,19 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
```
Comment (igaw) [inline tests/nvme_copy_test.py:1]: nvme-cli is license under GPL-2.0-only, libnvme is LGPL-2.1-or-later. I don't mind if you license this file under LGPL as it is compatible.

File: tests/nvme_copy_test.py [hunk 3]
```diff
@@ -1,17 +1,19 @@
...
 # This program is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
+# GNU Lesser General Public License for more details.
 #
-# You should have received a copy of the GNU General Public License
-# along with this program; if not, write to the Free Software
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
```
Comment (igaw) [inline tests/nvme_copy_test.py:16]: With the SPDX in place I'd say we also remove this redundant license text. For example the Linux kernel uses this form for Python code:

```python
#!/usr/bin/env python3
# SPDX-License-Identifier: GPL-2.0
"""
tdc_multibatch.py - a thin wrapper over tdc_batch.py to generate multiple batch
files

Copyright (C) 2019 Vlad Buslov <vladbu@mellanox.com>
"""

```

or

```python
#!/usr/bin/env python3
# SPDX-License-Identifier: GPL-2.0
#
# A thin wrapper on top of the KUnit Kernel
#
# Copyright (C) 2019, Google LLC.
# Author: Felix Guo <felixguoxiuping@gmail.com>
# Author: Brendan Higgins <brendanhiggins@google.com>


```

**Review by igaw — CHANGES_REQUESTED** [review]

**Comment by igaw** [comment]
FYI, I've squashed the changes together. This make the commit history a bit better to read.

All looks good. Thanks for updating the License info, excellent work.

**Review by igaw — APPROVED** [review]

**Comment by Arunpandian15** [comment]
Thank You :)

**Pull request merged**